specs/_features/maxeb_increase/capella.py

```diff
@@ -321,6 +323,8 @@ class Validator(Container):
     activation_epoch: Epoch
     exit_epoch: Epoch
     withdrawable_epoch: Epoch  # When validator can withdraw funds
+    # TODO: may compress into some other validator field
+    consolidated_to: ValidatorIndex
```

Comment:
From our Devconnect discussion:

If adding an extra field will become a concern, one of the alternatives is to rename `slashed` to e.g. `flags` and introduce `CONSOLIDATED_FLAG_INDEX` to use `activation_eligibility_epoch` (the field that is not used anywhere else other than assigning `activation_epoch`) as a `consolidated_to` if the flag is set.